### PR TITLE
rules from Nov 8

### PR DESCRIPTION
**Pull request by mlutze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -308,4 +308,44 @@ There may only be up to 1 dead pile, marked by the Death Token,
...
+After dealing, but before the round begins,
+the Dealer must distributed one card, face up, to each player,
+starting from the Dealer's left.
+This is the player's Salad.
+The player with the lowest (Aces low) Salad plays first.
+
+Play continues clockwise.
+
+As long as there is no player with the lowest top card in their Salad,
+the player with the top Salad card receives an additional card on their Salad.
```
Comment (JonathanStarup) [inline README.md]: you mean "with the lowest salad"?

File: README.md [hunk 2]
```diff
@@ -308,4 +308,44 @@ There may only be up to 1 dead pile, marked by the Death Token,
...
+- number cards
+- cards of a particular suit
+
+Aces count as both low and high for this rule, allowing sequences to "wrap around".
+
+## Salad Swap
+_Added by Jonathan, 8 Nov, 2023_
+
+Before the round begins,
+players may exchange cards from their hands with cards from their Salads.
```
Comment (JonathanStarup) [inline README.md]: at any point 😈

File: README.md (hunk 1 above)
Comment (mlutze) [reply README.md]: It was supposed to be "oldest top salad card"

File: README.md (hunk 2 above)
Comment (mlutze) [reply README.md]: seems confusing but we'll try :D

**Review by JonathanStarup — APPROVED** [review]

**Pull request merged**